doc/restcli.md

```diff
@@ -0,0 +1,15 @@
+# CLI REST endpoint
+
+Livepeer node exposes REST endpoint which is used by `livepeer_cli` for communication with node.
+By default it is exposed on port 7935.
```

Comment:
"By default, the CLI listens to `localhost:7935`. This can be adjusted with the `-cliAddr <interface>:<port>` flag."

We should also add a word of warning here about exposing the CLI to the world.

Reply:
fixed